client/src/components/Link/Link.test.js

```diff
@@ -1,40 +1,38 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 
-import Button from './Button';
+import Link from './Link';
 
-describe('Button', () => {
+describe('Link', () => {
   it('exists', () => {
-    expect(Button).toBeDefined();
+    expect(Link).toBeDefined();
   });
 
   it('basic', () => {
-    expect(shallow(<Button />)).toMatchSnapshot();
+    expect(shallow(<Link href="/#" />)).toMatchSnapshot();
```

Comment:
Throughout this file, let's just remove the addition of `href`, it's better to keep the tests aligned to mostly the current tests.